semgrep/semgrep/core_runner.py

```diff
@@ -476,7 +475,9 @@ def _run_rules_direct_to_semgrep_core(
             target_file_name, "w+"
         ) as target_file:
 
-            targets_with_rules = self._get_targets(rules, target_manager, all_targets)
+            targets_with_rules = self._create_core_run_planner(
+                rules, target_manager, all_targets
+            )
             target_file.write(json.dumps(targets_with_rules))
```

Comment:
```suggestion
            plan = self._plan_core_run(rules, target_manager, all_targets)
            target_file.write(json.dumps(plan))
```
this is in case you accept the above suggestion